app/views/bookmarks/_filters.html.erb

```diff
@@ -131,6 +131,12 @@
       <dd class="submit actions"><%= f.submit ts("Sort and Filter") %></dd>
     </dl>
 
+    <% if @pseud || @user || @collection || @tag %>
+    <p class="footnote">
```

Comment:
In most places in our views we indent 2 spaces inside a conditional, let's also do that here and in works/_filters.